spec/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner_spec.rb

```diff
@@ -80,4 +80,45 @@
       expect(hash[:full_data]).to be_an String
     end
   end
+
+  context "event_monitor" do
+    it "reset_event_monitor_handle" do
+      subject.send(:reset_event_monitor_handle)
+      expect(subject.instance_variable_get('@event_monitor_handle')).to be_nil
```

Comment:
But there is not output in this tests so I need to check if they are doing that we expect. Not?

Reply:
No, if we change the impl that would mean the tests would break. Think of the object as a black box, how does these changes affect the behaviour for an outside observer?

Reply:
Mmmm, Ok I can work with a double maybe, I'll change it. thanks